lean3/contribute/index.html

```diff
@@ -112,8 +112,8 @@ <h2 id="lifecycle-of-a-pr" class="markdown-heading">Lifecycle of a PR <a class="
 there should be a sidebar with panels "reviewers", "assignees", "labels", etc.
 Click on the "labels" header to add or remove labels from the current project.</p>
 <p>The most important labels are "awaiting-review" and "awaiting-author". If you label your PR with <strong>"awaiting-review"</strong>, someone will probably "review" it within a few days (depending on the size of the PR; smaller PRs will get quicker responses). The reviewer will probably leave comments and change the label to <strong>"awaiting-author"</strong>. You should address each comment, clicking the "resolve conversation" button once the problem is resolved. Ideally each problem is resolved with a new commit, but there is no hard rule here. Once all requested changes are implemented, you should change the label back to "awaiting-review" to start the process over again.</p>
-<p>After some iteration, a reviewer will "approve" the PR and the "ready-to-merge" label will be automatically applied to the PR. A bot called <code>bors</code> will take it from here. (See <a href="https://github.com/leanprover-community/mathlib/blob/master/docs/contribute/bors.md">here</a> for more detail about bors.)
-After responding appropriately to bors (if necessary), the PR will get added to the <a href="https://app.bors.tech/repositories/24316">"merge queue"</a>. The merge queue gets cleared automatically, but this takes some finite amount of time as it requires building branches of mathlib.</p>
+<p>After some iteration, a reviewer will "approve" the PR and the "maintainer-merge" label will be automatically applied to the PR. The PR is then posted to a private Zulip stream for maintainers to look at. Once a maintainer is happy with the PR, they too will "approve" it and the "ready-to-merge" label will be automatically applied to the PR. A bot called <code>bors</code> will take it from here. (See <a href="https://github.com/leanprover-community/mathlib/blob/master/docs/contribute/bors.md">here</a> for more detail about bors.)
```

Comment:
https://github.com/leanprover-community/mathlib/blob/master/docs/contribute/bors.md is outdated. I suggest we add in a later a new page on the website explaining how bors works.